### ISSUE TITLE
Use the kernel picker to allow user to select cluster/db

### ISSUE THREAD

**Issue by DonJayamanne** [DonJayamanne/vscode-kusto#18]
* Add a single item <Configure Kusto Connection>
* When the user selects this from the kernel quick pick, I'll:
	* Hook into the kernel selection changed event & see that the user chose this option
	* Now i'll display my own quick pick asking the user to select a cluster & db (using https://github.com/microsoft/vscode-extension-samples/blob/main/quickinput-sample/src/multiStepInput.ts#L7)
* Once user selects a cluster & database, I update the display name & description of the kernel to the real values & then I'll contribute a new kernel that'll show up as `<Configure Kusto Connection>`
* This way user can still use the kernel picker to add/configure new connections.

Basically its a hack to introduce my own kernel picker (hacky).

https://github.com/microsoft/vscode/issues/119949#issuecomment-812587384

This is more of a UX exploration.
